example/src/ModifyQuantity/ModifyQuantityPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import { ModifyQuantity, Screen, useTheme } from 'smartway-react-native-ui';
+
+export const ModifyQuantityPage = () => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        container: {
+            alignItems: 'center',
+            marginTop: theme.sw.spacing.m,
+        },
+    });
+
+    const [value, setValue] = useState<string>('0');
```

Comment:
I think we should use an int here: it better convey that this is a quantity and not a textual input.
This input will be used to modify a quantity, and the quantity will be stored as an int anyway.

Reply:
I will check that :)

Reply:
Convert it to string when you pass it to the TextInput value props

```ts
<TextInput value={parseInt(value)}/>
```